src/main/java/uk/gov/companieshouse/api/accounts/exception/handler/GlobalExceptionHandler.java

```diff
@@ -35,6 +37,15 @@
     @Value("${invalid.value}")
     public String invalidValue;
 
+    @ExceptionHandler(Exception.class)
+    protected ResponseEntity<Object> handleException(Exception ex) {
+        HashMap<String, Object> message = new HashMap<>();
+        message.put("message", ex.getMessage());
+        message.put("error", ex.getClass());
+        STRUCTURED_LOGGER.error(ex.getMessage(), message);
```

Comment:
We're going to be logging the exception message twice; once explicitly and once as part of `message`. Is this required?